Cargo.toml

```diff
@@ -25,6 +25,7 @@ parking_lot = "0.10.2"
 uuid = { version = "0.8.1", features = ["v4"] }
 rand = "0.7"
 hex = "0.4.2"
+serde-big-array = "0.3.0"
```

Comment:
Added to handle serialization of arrays with more than 32 elements (TCK: 2 + 32 + 32 = 66)

Reply:
Why 2+32+32?

Reply:
Ah, I see now. This is `TemporaryContactKey`, which has also index (2 bytes) and rvk (32 bytes).